app/db.js

```diff
@@ -16,6 +16,13 @@ const client = knex({
   },
 })
 
+async function addRecipe(recipe) {
+  return client('recipes')
+    .insert(recipe)
+    .returning(['id', 'externalId', 'name', 'imageAttachmentId', 'material', 'alloy', 'price', 'supplier'])
+}
```

Comment:
This can probably be generalised.

Reply:
this was a point I was trying to get to but we decided it might be better as a later refactor rather than guessing what it should be now